filesystem/iso9660.ksy

```diff
@@ -7,269 +7,959 @@ meta:
+                                doc-ref: rras 6d.0
+                                type: b1
+                      susp_ce:
+                        doc-ref: susp 5.1
+                        seq:
+                          - id: length
+                            type: u1
+                          - id: version
+                            type: u1
+                            valid: 1
+                          - id: ca_location
+                            type: u4bi
+                          - id: ca_offset
+                            type: u4bi
+                          - id: ca_length
+                            type: u4bi
+                      rrip_cl:
+                        doc-ref: rrip 4.1.5.1
+                        seq:
+                          - id: length
+                            type: u1
+                          - id: version
+                            type: u1
+                            valid: 1
+                          - id: lba_child
+                            type: u4bi
+                      susp_er:
+                        doc-ref: susp 5.5
+                        seq:
+                          - id: length
+                            type: u1
+                          - id: version
+                            type: u1
+                            valid: 1
+                          - id: len_id
+                            type: u1
+                          - id: len_des
+                            type: u1
+                          - id: len_src
+                            type: u1
+                          - id: ext_ver
+                            type: u1
+                          - id: ext_id
+                            size: len_id
+                          - id: ext_des
+                            size: len_des
+                          - id: ext_src
+                            size: len_src
+                      susp_es:
+                        doc-ref: susp 5.6
+                        seq:
+                          - id: length
+                            type: u1
+                          - id: version
+                            type: u1
+                            valid: 1
+                          - id: ext_seq
+                            type: u1
+                      rrip_nm:
+                        doc-ref: rrip 4.1.4
+                        seq:
+                          - id: length
+                            type: u1
+                          - id: version
+                            type: u1
+                            valid: 1
+                          - id: reserved
+                            doc: |
+                              Grouped all 4x reserved into a single reserved
+                            type: b5
+                          - id: parent
+                            type: b1
+                          - id: current
+                            type: b1
+                          - id: continue
```

Comment:
I would recommend changing this to `continued` as `continue` is a reserved keyword in Python and it is causing some issues.